hudi-common/src/main/java/org/apache/hudi/common/model/HoodieAvroIndexedRecord.java

```diff
@@ -47,40 +48,62 @@
  * This only use by reader returning.
  */
 public class HoodieAvroIndexedRecord extends HoodieRecord<IndexedRecord> {
+  private static final long serialVersionUID = 1L;
```

Comment:
k  this is a  HoodieRecord.. even though it says `...IndexedRecord` . problem for another day.

Reply:
Because we already have a `HoodieAvroRecord`, hmm...